### PR TITLE
gh-pages add showcase legend, rewrite resize code

### PR DESCRIPTION
**Pull request by 2xAA**
Adds a legend to show what the arrows represent in the demo.
Moves styles out to a stylesheet.
Rewrites resize code for succinctness.

### CODE REVIEW

**Comment by 2xAA** [comment]
screen shot 2017-04-17 at 17 40 24
Looks like this

File: src/main.js [hunk 1]
```diff
@@ -32,19 +31,19 @@
...
-  renderer.domElement.style.height = 'auto';
-  // window.addEventListener('resize', function(){
-  //   let canvasWidth = document.querySelector('.col').offsetWidth;
-  //   resolution = canvasWidth/16*10;
-  //   renderer.setSize(resolution * aspectRatio, resolution);
-  //   renderer.domElement.height = resolution * window.devicePixelRatio;
-  //   renderer.domElement.width = canvasWidth * window.devicePixelRatio;
-  // });
-  document.querySelector('#showcase').appendChild(renderer.domElement);
+  var renderer = new THREE.WebGLRenderer({ canvas: document.querySelector("canvas") });
```
Comment (hughrawlinson) [inline src/main.js]: I didn't know this was a thing, this solves so much shit!

File: src/main.js [hunk 2]
```diff
@@ -32,19 +31,19 @@
...
+
+
+  function resize() {
+    var resolution = renderer.domElement.parentNode.offsetWidth / 16 * 10;
+    renderer.setPixelRatio(window.devicePixelRatio ? window.devicePixelRatio : 1);
+    renderer.setSize(resolution * aspectRatio, resolution);
+    renderer.domElement.style.width = '100%';
```
Comment (hughrawlinson) [inline src/main.js]: could we do this with css rather than with js now that THREE is using an existing canvas element?

Comment (2xAA) [inline src/main.js]: Unfortunately not, THREE writes `style.width` and `style.height` to the Canvas when caling `setSize` which we need to overwrite :/

Comment (2xAA) [inline src/main.js]: In fact, we don't need to call setSize. We should be updating the projection matrix of the camera. Don't merge yet.

Comment (2xAA) [inline src/main.js]: setSize is required, but I've revised the way I'm setting the canvas size and it looks very crisp now.

Comment (hughrawlinson) [inline src/main.js]: from: https://threejs.org/docs/manual/introduction/Creating-a-scene.html

> If you wish to keep the size of your app but render it at a lower resolution, you can do so by calling setSize with false as updateStyle (the third argument). For example, setSize(window.innerWidth/2, window.innerHeight/2, false) will render your app at half resolution, given that your <canvas> has 100% width and height

Does that help?

**Comment by hughrawlinson** [comment]
@2xAA how are you getting around `getUserMedia() no longer works on insecure origins. To use this feature, you should consider switching your application to a secure origin, such as HTTPS. See https://goo.gl/rStTGz for more details.` in Chrome?

**Comment by 2xAA** [comment]
Oh, we can remove the `--host 0.0.0.0` in package.json if needed. I needed to test on mobile easily.
We can still go to `localhost:8000`

**Comment by hughrawlinson** [comment]
Oh perfect, leave the host in there then :)

**Pull request merged**